src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -659,22 +850,37 @@ public enum RisingRivals implements LogicCardInfo {
           pokeBody "Sticky Hold", {
             text "If Gastrodon East Sea is switched or retreats to your Bench, move as many Energy cards attached to Gastrodon East Sea as you like to the new Active Pokémon."
             delayedA {
+              after FALL_BACK, self, {
+                if(self.cards.energyCount(C) && confirm("Use Sticky Hold?")) {
+                  def energyList = self.cards.select(max:self.cards.filterByType(ENERGY).size(),"Move any number of Energy cards attached from $self to ${self.owner.pbg.active}",cardTypeFilter(ENERGY))
+                  energyList.each {
+                    energySwitch(self,self.owner.pbg.active,it)
```

Comment:
I recommend suppressing the energySwitch log and implementing a custom `bc` message to avoid spamming the log.